src/core/server/http/http_server.ts

```diff
@@ -75,6 +75,7 @@ export interface HttpServerSetup {
     isAuthenticated: AuthStateStorage['isAuthenticated'];
     getAuthHeaders: AuthHeadersStorage['get'];
   };
+  isTLSEnabled: boolean;
```

Comment:
I think this should be capitalized `isTlsEnabled` to be consistent with `HttpServer` and `SslConfig`

Reply:
Ok